dt/uni-beta/README.md

```diff
@@ -0,0 +1,11 @@
+# Deployed Topology - Beta
```

Comment:
shouldn't this file be taken from the general PR which holds all proposed uni topologies?

Reply:
Copied it from #45